packages/contracts-bedrock/scripts/deploy/DeployStandardValidator.s.sol

```diff
@@ -334,28 +277,10 @@ contract DeployStandardValidator is Script {
     }
 
     function assertValidValidator(DeployStandardValidatorInput _si, DeployStandardValidatorOutput _so) internal view {
-        address validator = _so.validator();
-
-        if (keccak256(bytes(_si.release())) == keccak256(bytes("v1.8.0"))) {
-            assertValidValidatorV180(_si, validator);
-        } else if (keccak256(bytes(_si.release())) == keccak256(bytes("v2.0.0"))) {
-            assertValidValidatorV200(_si, validator);
-        }
-    }
-
-    function assertValidValidatorV180(DeployStandardValidatorInput _si, address _validator) internal view {
-        IStandardValidatorV180 v180 = IStandardValidatorV180(_validator);
-        require(address(v180.superchainConfig()) == address(_si.superchainConfig()), "SV180-10");
-        require(v180.l1PAOMultisig() == _si.l1PAOMultisig(), "SV180-20");
-        require(v180.challenger() == _si.challenger(), "SV180-40");
-        require(v180.withdrawalDelaySeconds() == _si.withdrawalDelaySeconds(), "SV180-50");
-    }
-
-    function assertValidValidatorV200(DeployStandardValidatorInput _si, address _validator) internal view {
-        IStandardValidatorV200 v200 = IStandardValidatorV200(_validator);
-        require(address(v200.superchainConfig()) == address(_si.superchainConfig()), "SV200-10");
-        require(v200.l1PAOMultisig() == _si.l1PAOMultisig(), "SV200-20");
-        require(v200.challenger() == _si.challenger(), "SV200-40");
-        require(v200.withdrawalDelaySeconds() == _si.withdrawalDelaySeconds(), "SV200-50");
+        IStandardValidator validator = IStandardValidator(_so.validator());
+        require(address(validator.superchainConfig()) == address(_si.superchainConfig()), "SV300-10");
```

Comment:
do the SV300 error strings still make sense now that the SV isn't tied to a version per se?

Reply:
fixed that here: f66a04e68a10da2b44237775d30915e096cdd046